src/json.ts

```diff
@@ -490,130 +509,193 @@ export function getJsonRpcIdValidator(options?: JsonRpcValidatorOptions) {
 export function validateJsonAndGetSize(
   jsObject: unknown,
   skipSizingProcess = false,
-): [isValid: boolean, plainTextSizeInBytes: number] {
+): ValidatedJson {
```

Comment:
This function is returning the `value: unknown` directly and this is an exported function - perhaps relax the return type to reflect reality or add some type guards in it?

Reply:
Never mind; got confused by the indentation in the GH diff.